### PR TITLE
Test- login redirect

### PR DESCRIPTION
**Pull request by trexdinosaur**
WIP

### CODE REVIEW

File: api/tests.py [hunk 1]
```diff
@@ -44,14 +46,54 @@ def setUp(self):
                                 "terminal"], timestamp=self.demo["timestamp"],
                             token=self.demo["token"],
                             status=self.demo["status"])
-
+    def test_get_redir_user_demo(self):
+        self.demo = {
```
Comment (AvaisP) [inline api/tests.py]: This is already there in setUp right?

**Comment by trexdinosaur** [comment]
Fixed @AvaisP, also I sent you a question over Gitter!

File: api/tests.py [hunk 2]
```diff
@@ -44,14 +46,40 @@ def setUp(self):
...
+        first_url, first_response = response.redirect_chain[0]
+        self.assertEqual(first_url, "/login?status=passed&token=test_token&username=testname&user_id=1001")
     def test_get_all_user_demos(self):
         response = self.client.get('/api/demo/user/%d' %
                                    (self.demo["user_id"]))
         responses = json.loads(response.content.decode('utf-8'))
         response = responses[0]
         self.assertEqual(response["id"], self.demo["id"])
         self.assertEqual(response["user_id"], self.demo["user_id"])
+        response = self.client.get('/api/demo/user/%d' %
```
Comment (AvaisP) [inline api/tests.py]: What is this supposed to do?

File: api/tests.py [hunk 3]
```diff
@@ -207,7 +235,7 @@ def setUp(self):
             props=self.input_component["props"],
             user_id=self.input_component["user_id"],
             demo=self.input_component["demo"])
-
+
```
Comment (AvaisP) [inline api/tests.py]: Remove the extra tab

**Comment by AvaisP** [comment]
Please resolve the conflicts and make the suggested changes @trexdinosaur

File: api/tests.py (hunk 2 above)
Comment (trexdinosaur) [reply api/tests.py]: It tests an exception

**Comment by trexdinosaur** [comment]
@AvaisP Would I still need to test all exceptions for the GCI task?

**Comment by AvaisP** [comment]
No @trexdinosaur 
Just resolve the conflicts

**Comment by trexdinosaur** [comment]
@AvaisP should be resolved! I will add some more tests in a few hours, but if possible, please accept the GCI Task!

**Comment by trexdinosaur** [comment]
@AvaisP, tests are running successfully on my localhost, how do i force a travis rebuild?

**Comment by AvaisP** [comment]
@trexdinosaur Seems like webpack-stats-local.json has been deleted. Please commit your local copy

**Comment by trexdinosaur** [comment]
@vpn1997 @AvaisP looks like #106 got rid of webpack stats and that's what's causing the issue....

**Comment by trexdinosaur** [comment]
@AvaisP it has been pushed!

**Pull request merged**